model/Core/Classes/Element.md

```diff
@@ -45,10 +45,10 @@ and inter-relatable content objects.
   - maxCount: 1
 - verifiedUsing
   - type: IntegrityMethod
-- externalReferences
+- externalReference
```

Comment:
This is called `externalRef` in SPDX 2.X.  Although there has been an argument that `externalReference` is clearer, I don't think that justification is enough to cause migration effort.  Note that the choice of `externalRef` as the name in 2.X was chosen after a discussion and comparison to other terms such as `externalReference`.  I added issue #234 to track renaming this back to the 2.3 value.

Reply:
I'd rather wait until the issue is resolved before going ahead and already changing it prior to consensus.